src/ArachneTest.cc

```diff
@@ -33,15 +33,13 @@ using CoreArbiter::MockSyscall;
 
 extern bool disableLoadEstimation;
 
+extern bool useCoreArbiter;
+
 extern std::atomic<uint32_t> numActiveCores;
 extern volatile uint32_t minNumCores;
 extern int* virtualCoreTable;
 
-#ifndef NO_ARBITER
 extern CoreArbiterClient& coreArbiter;
```

Comment:
Our coding style is not to use refs unless they are unavoidable (e.g., in some templating situations). It doesn't look like you created this ref, but it would be good to eliminate them when you see them. It looks like you have already been doing that elsewhere? Perhaps this declaration is no longer needed?